### DISCUSSION TITLE
Why did caddy suddenly go away as proxy server?

### DISCUSSION THREAD

**Discussion by weholt** [cookiecutter/cookiecutter-django#3636] [General]
The last time I tried the django cookiecutter recipe Caddy was used as proxy, but now Nginx is used instead. Why? Are there any issues using Caddy in production serving django apps we should be aware of?

**Comment by browniebroke** [comment]
We're not using nginx, we're using Traefik. It has been like this for around 3 years: https://github.com/cookiecutter/cookiecutter-django/pull/1714

You can read through the issue and pull request to understand the rationale of the decision at the time

**Reply by weholt** [reply]
Oh, my bad. Thanks for the reply. I'll dive into the pull request :-)
